Makefile

```diff
@@ -9,23 +9,45 @@ TEST_RD_USER ?= `whoami`
 TEST_RD_HOSTNAME ?= localhost
 TEST_RD_URL="redis://$(TEST_RD_USER)@$(TEST_RD_HOSTNAME):6379/$(TEST_RD_NAME)"
 
-install:
+PG_FIX_CANDIDATES=./protos/node/node.pb.go \
+			./protos/status/status.pb.go \
+			./protos/escrow/escrow.pb.go \
+
+install: brew trongogo
+
+brew:
 	brew install protobuf
 	brew install prototool
 	brew install postgresql go
 	brew install redis go
 	brew services start postgresql
 	brew services start redis
 
+trongogo:
+	cd ../ && git clone https://github.com/TRON-US/protobuf
+	cd ../protobuf && make
```

Comment:
suggest a tech debt ticket to publish protobuf on brew, or somewhere where we can just install a binary